docs/index.md

```diff
@@ -179,6 +179,7 @@ lockFilePath: path/to/lock.file
 # See the helm usage (helm SUBCOMMAND -h) for more info on default values when those flags aren't provided.
 helmDefaults:
   kubeContext: kube-context          #dedicated default key for kube-context (--kube-context)
+  namespace: default                 #dedicated default key for namespace (--namespace)
```

Comment:
I think the default value shouldn't always be default.

It should be actually omitted by default.

When omitted, the namespace set in the kube-context will be used.

In case no namespace is set in the context, then `default` is used.

Reply:
your are correct, so in every go-code i put `""` as an empty string.
How should i document it, to clearify this behavour?